.github/workflows/build-natives.yaml

```diff
@@ -0,0 +1,57 @@
+name: Build Natives
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        include:
+        - os: windows-latest
+          vcvars: C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvarsall.bat
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Maven Version Info
+        run: mvn -version
+      - if: matrix.os != 'windows-latest'
+        name: OpenSSL Library Version Info
+        run: openssl version -v
+      - if: matrix.os == 'windows-latest'
+        name: Install Full OpenSSL Library
+        run: |
+          vcpkg install openssl
```

Comment:
Do you think it would be possible to output the OpenSSL Library Version Info after installing it by running `openssl version -v` just so we have a sanity check that the version installed is the default version that's getting used for building?